wallet/wallet-controller/src/lib.rs

```diff
@@ -123,6 +123,18 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static> Controller<T> {
         Ok(wallet)
     }
 
+    pub fn encrypt_wallet(&mut self, password: &Option<String>) -> Result<(), ControllerError<T>> {
```

Comment:
Please add documentation to those three function, including what the parameters mean. Given the controller is the heighest level function in the wallet, it's important to make everything clear so that a user who doesn't have to deal with the details of the wallet would understand how to use it.

Reply:
done